pontoon/base/views.py

```diff
@@ -607,6 +607,10 @@ def update_translation(request):
     except UserProfile.DoesNotExist as error:
         use_ttk_checks = True
 
+    # Disable checks for demo project.
+    if project.slug == 'demo':
+        use_ttk_checks = False
+
```

Comment:
Because we don't want users to be told to fix their comment during the step where they're asked to enter a translation. It doesn't actually matter at that point. But maybe we want that to happen only during the tour? That would be a lot more difficult to do though.

Reply:
OK, makes sense. I don't think we need to add further complexity.